index.js

```diff
@@ -3638,15 +3638,21 @@
   //.
   //. > S.get(S.is(Number), 'x', {})
   //. Nothing
+  //.
```

Comment:
`'purple'` is *meant* to satisfy the [Accessible][1] constraint but you are right to point out that it does not. It just occurred to me that we might be able to remove the Accessible constraint by applying `Object` to arguments in a few places. I'll investigate this possibility.


[1]: https://github.com/sanctuary-js/sanctuary/tree/v0.13.2#accessible-pseudotype